api/src/main/java/cz/muni/fi/dto/DriverCreateDTO.java

```diff
@@ -0,0 +1,110 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package cz.muni.fi.dto;
+
+import cz.muni.fi.enums.DrivingSkill;
+import java.util.Objects;
+
+/**
+ * @author Lucie Kurečková, 445264
```

Comment:
Please don't use diacritics without resolving problems that could occure, not even in javadocs